libres/lib/analysis/modules/ies_enkf.cpp

```diff
@@ -927,18 +920,18 @@ void *ies_enkf_get_ptr(const void *arg, const char *var_name) {
 
 analysis_table_type LINK_NAME = {
     .name = "IES_ENKF",
-    .initX = NULL,
     .updateA = ies_enkf_updateA,
+    .initX = NULL,
     .init_update = ies_enkf_init_update,
     .complete_update = NULL,
-    .alloc = ies_enkf_data_alloc,
     .freef = ies_enkf_data_free,
-    .has_var = ies_enkf_has_var,
+    .alloc = ies_enkf_data_alloc,
```

Comment:
Why are these changing place?

Reply:
Because the C++ compiler complains that they are not in the same order as defined in `analysis_table.hpp`.
C-compiler did not care.